rainier-test/src/test/scala/com/stripe/rainier/compute/CholeskyTest.scala

```diff
@@ -0,0 +1,24 @@
+package com.stripe.rainier.compute
+
+import com.stripe.rainier.sampler.RNG
+
+class CholeskyTest extends ComputeTest {
+    val rng = RNG.default
+    def testRandomSolve(): Unit =  {
+        val rank = rng.int(5) + 2
```

Comment:
nit: inconsistent spacing. In some places you use non-standard 4 (like here) but others you use 2 (below in ComputeTest).

Reply:
I normally just let scalafmt deal with that, I must have pushed here before compiling